docs/website/data/integrations.yaml

```diff
@@ -600,7 +600,20 @@ integrations:
       - https://github.com/StyraInc/opa-kafka-plugin
     inventors:
       - redhat
-
+  
+  ccbr:
+    title: ccbr 
+    description: CCBR is a policy management system project. It uses the policy language Rego to implement the CIS benchmark test of cloud native kubernetes. In addition, it integrates gatekeeper, manages its constraint templates, constraints and policies, and supports policy deployment and audit inspection.
+    software:
+      - ccbr
+    labels:
+      category: ZTA
+      layer: Policy Management
+    code:
+      - https://github.com/WirelessSecurityLab/ccbr
+    inventors:
+      - WirelessSecurityLab
```

Comment:
Make this lower case  and then add an entry for `wirelesssecuritylab` under the `organizations` section.

Reply:
@ashutosh-narkar https://github.com/open-policy-agent/opa/pull/5088